### PR TITLE
fix(doc): Wrong oneup service name

### LINKED ISSUE #1198: Incorrect documentation for integrating oneup/flysystem-bundle
### Bug Report

|    Q            |   A
|---------------- | ------
| BC Break        | ~yes~ no
| Bundle version  | 1.17.0 
| Symfony version | 5.2.6
| PHP version     | 7.4.16

#### Summary
Current documentation for integrate oneup/flysystem-bundle will lead to error exception when uploading an image.

#### Current behavior

When i upload image it give me an error as below:

```php
Service "default_filesystem" not found: the container inside "Symfony\Component\DependencyInjection\Argument\ServiceLocator" is a smaller service locator that only knows about the "oneup_flysystem.default_filesystem_filesystem" service.
```

#### How to reproduce
This is my `vich_uploader.yaml` :

```yaml
vich_uploader:
    db_driver: orm
    storage: flysystem

    mappings:
       file_upload:
           uri_prefix: /var/storage/local
           upload_destination: default_filesystem # I think because of this one
           namer:
                service: Vich\UploaderBundle\Naming\HashNamer
                options: { algorithm: 'sha512', length: 128 }
           directory_namer:
                service: Vich\UploaderBundle\Naming\CurrentDateTimeDirectoryNamer
                options:
                    date_time_format: 'Y/m/d' # will create directory "2018/23/09" for current date "2018-09-23"
                    date_time_property: uploadedAt # see above example
           inject_on_load: true
```

And this is `oneup_flysystem.yaml`:

```yaml
# Read the documentation: https://github.com/1up-lab/OneupFlysystemBundle/tree/master/Resources/doc/index.md
oneup_flysystem:
    adapters:
        default_adapter:
            local:
                location: '%kernel.project_dir%/var/storage/local'
    filesystems:
        default_filesystem:
            adapter: default_adapter
            mount: default_filesystem
```

#### Expected behavior

Successfully upload image.

### PR DESCRIPTION
**Pull request by Tofandel**
Fixes #1198

### CODE REVIEW

**Comment by garak** [comment]
Please add your comment to discussion in #1044

**Comment by Tofandel** [comment]
@garak This has absolutely nothing to do with relative path, the container name in the doc is wrong and plainly doesn't work if you follow the doc

See https://github.com/1up-lab/OneupFlysystemBundle/blob/main/doc/filesystem_create.md#create-and-use-your-filesystems

**Comment by garak** [comment]
> The service name must be `oneup_flysystem.%adapter_name%_filesystem`

That's true only in version 4. The current docs are correct for version 3.

File: docs/storage/flysystem.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ vich_uploader:
     mappings:
         product_image:
             uri_prefix: /images/products
-            upload_destination: product_image_fs
```
Comment (garak) [inline docs/storage/flysystem.md:98]: This change is incompatible with OneupFlysystemBundle version 3.
We need ti provide a valid solution for both versions

**Review by garak — CHANGES_REQUESTED** [review]

File: docs/storage/flysystem.md (hunk 1 above)
Comment (Tofandel) [reply docs/storage/flysystem.md:98]: Does this do it?

Since v4 is the default now we can leave the v3 way but as a comment

Comment (garak) [reply docs/storage/flysystem.md:98]: Yes

**Review by garak — CHANGES_REQUESTED** [review]
Can you solve conflict please?

**Review by garak — APPROVED** [review]

**Pull request merged**